appium/webdriver/webdriver.py

```diff
@@ -691,6 +691,24 @@ def battery_info(self):
         """
         return self.execute_script('mobile: batteryInfo')
 
+    @property
+    def events(self):
+        """ Retrieves event timing information from the current session
+        Usage:
+            events = driver.events
+
+        Returns:
+            `dict containing event timimgs information from the current session`
```

Comment:
typo: timimgs -> timings

And you can describe like below.

`dict: containing event timings information from the current session`

e.g. 
https://github.com/appium/python-client/blob/master/appium/webdriver/extensions/screen_record.py#L103-L106